src/m3ninx/index/regexp.go

```diff
@@ -28,6 +28,19 @@ import (
 	fstregexp "github.com/m3db/m3/src/m3ninx/index/segment/fst/regexp"
 )
 
+var (
+	// DotStartCompiledRegex is a CompileRegex that matches any input.
+	DotStarCompiledRegex CompiledRegex
```

Comment:
Does this need to be exported?  And if so, would you be opposed to making this a method that returns the initialized var?

i.e.
```go
var (
  dotStarCompiledRegex CompiledRegex
)

func DotStarCompiledRegex() CompiledRegex {
  return dotStarCompiledRegex
}
```

Reply:
done